src/input-multi-select/input-multi-select.tsx

```diff
@@ -218,6 +218,7 @@ export const InputMultiSelect = <T, V>({
                     disabled={disabled}
                     expanded={showOptions}
                     listboxId={internalId}
+                    popupRole="listbox"
```

Comment:
out of curiousity, what does this prop do?

Reply:
this translates to the `aria-haspopup` [role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-haspopup) which is used in combobox style of inputs to hint to accessibility tools that there is dropdown content

it should match the `role` of the dropdown content, in the case of nested select input, it should be `tree`, for basic lists, it should be `listbox`